src/daft-dashboard-client/README.md

```diff
@@ -0,0 +1,3 @@
+# Daft Dashboard Client
+
+To do!
```

Comment:
Why is this moved into `src`? I think `src` contains all the Rust code, shouldn't this remain in `dashboard/*`?

Reply:
Sure, I can move it back.

Reply:
Moved back to `/dashboard`.